<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>399</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="StartMenu">
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>399</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>127</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>SETTINGS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>300</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>QUIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>214</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>HALL OF FAME</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PB_launchGame">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>PLAY</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="GameWidget"/>
    <widget class="QWidget" name="Settings">
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>399</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>981</x>
         <y>340</y>
         <width>210</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>START MENU</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>10</y>
         <width>120</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 50px;</string>
       </property>
       <property name="text">
        <string>KEYS</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>150</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 30px;</string>
       </property>
       <property name="text">
        <string>LEFT</string>
       </property>
      </widget>
      <widget class="QLabel" name="keyL2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>150</y>
         <width>82</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>  ARROW_LEFT</string>
       </property>
      </widget>
      <widget class="QLabel" name="keyR2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>220</y>
         <width>85</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string> ARROW_RIGHT</string>
       </property>
      </widget>
      <widget class="QLabel" name="keyJ2">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>290</y>
         <width>82</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>    ARROW_UP</string>
       </property>
      </widget>
      <widget class="QLabel" name="keyL1">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>150</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="keyR1">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>220</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="keyJ1">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>290</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="editL">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>150</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editR">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>220</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editJ">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>290</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>EDIT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>220</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 30px;</string>
       </property>
       <property name="text">
        <string>RIGHT</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>290</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 30px;</string>
       </property>
       <property name="text">
        <string>JUMP</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="HOF">
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>399</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>435</x>
         <y>10</y>
         <width>330</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font-size: 50px;</string>
       </property>
       <property name="text">
        <string>HALL OF FAME</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>981</x>
         <y>340</y>
         <width>210</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 10px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>START MENU</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="EscMenu">
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1200</width>
        <height>399</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(0,0,0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>50</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>RESUME</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>175</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>MENU</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>299</y>
         <width>500</width>
         <height>60</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color:rgb(255, 255, 255);
	border:none;
	color:rgb(0, 0, 0);
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(196, 196, 196);
}
QPushButton:pressed {
	background-color:rgb(117, 117, 117);
}
</string>
       </property>
       <property name="text">
        <string>QUIT</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
